test/integration/lib/MockASE.ts

```diff
@@ -0,0 +1,49 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import {
+  AuthenticatedClient,
+  createAuthenticatedClient
+} from '@interledger/open-payments'
+import { AccountProvider, setupFromSeed } from 'mock-account-servicing-lib'
+import { createApolloClient } from './apolloClient'
+import { WebhookServer } from './WebhookServer'
+import { TestConfig } from './config'
+
+/** Mock Account Servicing Entity */
+export class MockASE {
+  private config: TestConfig
+  private apolloClient: ApolloClient<NormalizedCacheObject>
+  public accounts: AccountProvider
+  public opClient!: AuthenticatedClient
+  public webhookServer: WebhookServer
+
+  // Use .create factory because async construction
+  public static async create(config: TestConfig): Promise<MockASE> {
+    const mase = new MockASE(config)
+    await mase.initAsync()
+    return mase
+  }
+
+  // Private to ensure it doesnt get called directly.
+  // Use static MockASE.create instead.
+  private constructor(config: TestConfig) {
+    this.config = config
+    this.apolloClient = createApolloClient(config.graphqlUrl)
+    this.accounts = new AccountProvider()
+    this.webhookServer = new WebhookServer(this.apolloClient, this.accounts)
+    this.webhookServer.start(this.config.webhookServerPort)
+  }
+
+  private async initAsync() {
+    await setupFromSeed(this.config, this.apolloClient, this.accounts)
+
+    this.opClient = await createAuthenticatedClient({
+      privateKey: this.config.key,
+      keyId: this.config.keyId,
+      walletAddressUrl: this.config.walletAddressUrl
+    })
```

Comment:
Not quite right. Getting an invalid signature error from auth's grant initialization middleware. 

The `keyId` and `walletAddressUrl` are coming from uncommitted environment variables. They are `rafiki` and `https://cloud-nine-wallet-test-backend/.well-known/pay` (for the coud nine client) respectively. The key is loaded from this file (same key as from localenv): https://github.com/interledger/rafiki/pull/2380/files#diff-059e91a06656fac95dd3ee765c22dc8b3ae16d4e41074ce510a111bce7bd3f59

I'm not sure the `keyId` is right but using one of the `keyId`'s formed in `setupFromSeed` (`keyid-{account.number}`) gave me `invalid client` which happens earlier on in the auth middleware where it cant find the key by the `client` and `keyId`.

Reply:
This ended up being fine, but I needed to use the same private key file everywhere (here in the test and in each backend service).